operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -698,17 +729,56 @@ private void addIncludesSearchParameters(String compartmentMemberType, Multivalu
     
     private void addSearchParameterIfNotExcluded(String compartmentMemberType, String code, 
         ResourceType.Value subResourceType, MultivaluedMap<String, String> searchParameters, 
-        List<String> allowedIncludes) {
+        List<String> allowedIncludes, List<String> extraResources) {
```

Comment:
if you can shift this right by 4 (same with 731), it's the project style.

Reply:
Done